components/more-less/test/more-less.test.js

```diff
@@ -6,7 +6,7 @@ export function waitForHeight(elem) {
 	return new Promise((resolve) => {
 		function check() {
 			const content = elem.shadowRoot.querySelector('.d2l-more-less-content');
-			if (content.style.height === '') {
+			if (content.style.maxHeight === content.scrollHeight) {
```

Comment:
Oops my mistake. I think this test is just checking if the attribute has been initialized yet. So I think we want:
`if (content.style.maxHeight === '') {`

Reply:
Change has been made, thanks!